node/status_node.go

```diff
@@ -551,3 +553,14 @@ func (n *StatusNode) ensureSync(ctx context.Context) error {
 		}
 	}
 }
+
+// discoveryTopics calculates the discovery topics to be acrive based on the
+// different features active on the node.
+func (n *StatusNode) discoveryTopics() ([]discv5.Topic, map[discv5.Topic]params.Limits) {
```

Comment:
@adambabik is ok to discover mail servers whenever discovery is active, or we need an explicit option to enable it?
I guess we could avoid this logic by simply adding `MailServerDiscoveryTopic` and its limits to `RequireTopics` and `RegisterTopics`, so, you'll be forced to explicitly enable it through the config.